lib/links/parser.rb

```diff
@@ -0,0 +1,65 @@
+require 'httparty'
+
+class Parser
+
+  @@elements = { a: ['href'],
+                img: ['src'],
+                script: ['src'],
+                frame: ['src'],
+                iframe: ['src'],
+                link: ['href'],
+                source: ['src', 'srcset'],
+                track: ['src'] }
+
+  attr_reader :url
+  #attr_accessor :doc
```

Comment:
If it's commented out, it can be removed 😄

Reply:
Oops, I let this one slip